src/store/toast-store.js

```diff
@@ -72,4 +72,9 @@ store.registerHandler('INVALID_CODE', () => {
     store.emitChange();
 });
 
+store.registerHandler('MODEL_DELETED', model => {
+    addToast(`${model.name}(${model.address}) was deleted`);
```

Comment:
Nit: I think it would look better to have a space between `${model.name}` and `(${model.address})`.